src/ApiService/ApiService/OneFuzzTypes/Requests.cs

```diff
@@ -264,6 +264,13 @@ public record PoolCreate(
     [property: Required] Os Os,
     [property: Required] Architecture Arch,
     [property: Required] bool Managed,
+    Guid? ObjectId = null,
+    bool Update = false
```

Comment:
Why do we need this?

Reply:
This was just missing from the data returned from the server. It is the object id of the principal used by an unmanaged node